src/test/java/de/mediathekview/mserver/crawler/orf/tasks/OrfArchiveLetterPageTaskTest.java

```diff
@@ -0,0 +1,45 @@
+package de.mediathekview.mserver.crawler.orf.tasks;
+
+import de.mediathekview.mserver.crawler.orf.OrfTopicUrlDTO;
+import de.mediathekview.mserver.crawler.orf.OrfConstants;
+import de.mediathekview.mserver.testhelper.JsoupMock;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import org.hamcrest.Matchers;
+import org.jsoup.Jsoup;
+import static org.junit.Assert.assertThat;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({Jsoup.class})
+public class OrfArchiveLetterPageTaskTest {
+
+  private final OrfTopicUrlDTO[] expectedUrls= new OrfTopicUrlDTO[] {
+    new OrfTopicUrlDTO("Best of \"ZIB 2\"-Interviews", "http://tvthek.orf.at/archive/Best-of-ZIB-2-Interviews/7874678"),
+    new OrfTopicUrlDTO("Bundesheer-Archiv", "http://tvthek.orf.at/archive/Bundesheer-Archiv/5106911"),
+    new OrfTopicUrlDTO("Bundespräsidentenwahlen in Österreich", "http://tvthek.orf.at/archive/Bundespraesidentenwahlen-in-Oesterreich/13304953"),
+    new OrfTopicUrlDTO("Bundestagswahl-Archiv", "http://tvthek.orf.at/archive/Bundestagswahl-Archiv/6524565"),
+    new OrfTopicUrlDTO("Die Geschichte des Burgenlands", "http://tvthek.orf.at/archive/Die-Geschichte-des-Burgenlands/9236430"),
+    new OrfTopicUrlDTO("Die politische Geschichte der Zweiten Republik", "http://tvthek.orf.at/archive/Die-politische-Geschichte-der-Zweiten-Republik/9501692")
+  };
+  
+  @Test
+  public void test() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)